src/Components/Dot/index.jsx

```
@@ -1,24 +1,24 @@
-	);
-};
-
-Dot.propTypes = {
-	color: PropTypes.string,
-	size: PropTypes.string,
-};
-
-export default Dot;
+import PropTypes from "prop-types";
+
+const Dot = ({ color = "gray", size = "4px", style }) => {
```

ESLint

[error] 'style' is missing in props validation

(react/prop-types)